ng/manage.ts

```diff
@@ -1473,11 +1473,18 @@ angular.module("armExplorer", ["ngRoute", "ngAnimate", "ngSanitize", "ui.bootstr
 function getPowerShellFromResource(value: ISelelctHandlerReturn, actions: IAction[]): string {
     var returnString = "# PowerShell equivalent script\n\n";
     var resourceInfo = GetResourceTypeAndName(value);
+    var children = value.resourceDefinition.children;
 
     // add GET related cmdlet if available
+    var getCmdlet = "Get-AzureRmResource ";
+    var apiVersion = value.resourceDefinition.apiVersion;
     if (value.httpMethod.toLowerCase().indexOf("get") != -1) {
         returnString += "# GET " + GetActionName(value.url) + "\n";
-        returnString += "Get-AzureRmResource " + resourceInfo + " -ApiVersion " + value.resourceDefinition.apiVersion + "\n\n";
+        if (typeof children === "string" && !Array.isArray(children) && value.url.split("/").length > 7) {
```

Comment:
The `> 7` part needs a comment (by example), as it's hard to see where it comes from.